solr/solr-ref-guide/src/_includes/head.html

```diff
@@ -11,6 +11,7 @@
 <link rel="stylesheet" href="{{ "css/ref-guide.css" }}">
 <link rel="stylesheet" href="{{ "css/decoration.css" }}">
 <link rel="stylesheet" href="{{ "css/search.css" }}">
+<link rel="stylesheet" href="{{"css/rouge-style-github.css" }}"
```

Comment:
Is there no way to do this without explicitly adding the CSS? It adds to the items that we must manage and will require us to maintain this CSS over time.

Reply:
As above, you can make inline styles, but the file sizes grow. But I can switch to inline if that's such a significant concern.